src/synchronizeContacts.js

```diff
@@ -87,6 +87,10 @@ const shouldCreateOnGoogle = (cozyContact, contactAccountId) => {
   )
 }
 
+const shouldDeleteOnGoogle = cozyContact => {
+  return cozyContact.trashed
+}
```

Comment:
when I did the delete the other way, I thought that maybe we could replace those 6 functions by 2 functions (one for cozy loop and for the google loop) that returns a string `shouldCreate` | `shouldUpdate` | `shouldDelete` ? What do you think?

Reply:
That could be good yes. I'll have a look at the whole code again and see what makes sense.

Reply:
I did it in 1f8ef2164bc9b6bfa978992c16512369e5a35235 although now that it's done I'm not sure it such a good idea. It doesn't make the structure inside the main loops any lighter, and instead of 6 very purposeful and explicit function, we have to umbrella functions where it's harder tu understand what's going on, I think.